internal/application/billing/templates/bill_receipt.html

```
@@ -0,0 +1,219 @@
+{{define "bill_receipt.html"}}<!doctype html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)